src/dawn_native/vulkan/DeviceVk.cpp

```diff
@@ -421,13 +421,16 @@ namespace dawn_native { namespace vulkan {
             extensions->push_back(extension);
         };
 
-        // vktrace works by instering a layer, so we need to explicitly enable it if it is present.
+        // vktrace works by instering a layer, but we hide it behind a macro unless we are
```

Comment:
It would be nice to explain why, for exmaple: The vktrace layer crashes when used without the vktrace server started, see this vktrace issue <LINK TO THE ISSUE>